models/mpas_atm/model_mod.f90

```diff
@@ -1226,15 +1254,15 @@ subroutine model_interpolate(state_handle, ens_size, location, obs_type, expecte
    end select
 endif
 
-if (debug > 10 .and. do_output()) &
+if (debug > 1 .and. do_output()) &
```

Comment:
i'm not sure we want to drop the debug level on this and the next test down to 1.  i think they were ok as 10 and 4.

Reply:
un-dropped the debug level